src/tools/miri/src/lib.rs

```diff
@@ -144,7 +144,7 @@ pub const MIRI_DEFAULT_ARGS: &[&str] = &[
     "-Zmir-emit-retag",
     "-Zmir-keep-place-mention",
     "-Zmir-opt-level=0",
-    "-Zmir-enable-passes=-CheckAlignment",
+    "-Zub-checks=no",
```

Comment:
That also disables library UB checks, which is not what we want I think?

Reply:
Ah. Yes. I confused myself thinking about the other kinds of UB checks I want to land. No change is needed here.